docs/docker_deployment.md

```diff
@@ -91,3 +91,27 @@ different exporters, you may find them and their documentation available at
 After updating the `otelcol-config-extras.yml`, start the demo by running
 `docker compose up`. After a while, you should see the traces flowing into
 your backend as well.
+
+## Vendor Forking
+
+Setting up a fork or a demo usually only requires overriding some environment
+variables and possibly replacing some container images.
+
+Live demos can be added to the [README](https://github.com/open-telemetry/opentelemetry-demo/blob/main/README.md?plain=1#L186)
+
+### Configuring the Collector
```

Comment:
The content of this chapter is already explained above, so I think this chapter can be removed completely.